.circleci/config.yml

```diff
@@ -325,19 +325,28 @@ jobs:
       - run: git submodule init && git submodule update
       - run: mkdir -p /tmp/workspace/
       - run:
-          name: "Extract commit message"
+          name: Extract commit message
           command: git log --format=%B -n 1 $CIRCLE_SHA1 > /tmp/workspace/commit-message
-      - run: PUPPETEER_SKIP_DOWNLOAD=true PUPPETEER_SKIP_CHROMIUM_DOWNLOAD=true npm ci
-      - run: npm run build
       - run:
-          name: "Remove files we don't want in the zip file."
+          name: Install npm dependencies
+          command: PUPPETEER_SKIP_DOWNLOAD=true PUPPETEER_SKIP_CHROMIUM_DOWNLOAD=true npm ci
+      - run:
+          name: Build assets
+          command: npm run build
+      - run:
+          name: Remove unnecessary files
           command: rm -rf .circleci .git .githooks assets/src bin tests
-      # Exclude node_modules instead of removing, which takes a long time (lots of small files).
       - run:
-          name: "Create zip file"
+          name: Inject version to composer
+          command: |
+            apk add jq
+            cat composer.json | jq --arg version ${CIRCLE_TAG} '. + {version: $version}' > tmp.json
+            cp -f tmp.json composer.json
```

Comment:
I just realized tmp.json ends up in the final zip

Reply:
If it gets immediately overwritten in the next statement, we could also do 
```bash
cat composer.json | jq --arg version ${CIRCLE_TAG} '. + {version: $version}' > composer.json

Reply:
Using `mv tmp.json composer.json` would fix that